tests/test_MultilabelBinaryPolicy.py

```diff
@@ -0,0 +1,794 @@
+    ExclusivePolicy,
+    LessExclusivePolicy,
+    ExclusiveSiblingsPolicy,
+    InclusivePolicy,
+    LessInclusivePolicy,
+    SiblingsPolicy,
+)
+
+fixtures_loc = Path(__file__).parent / "fixtures"
+
+
+@pytest.fixture
+def digraph():
+    return nx.DiGraph(
+        [
+            ("r", "1"),
+            ("r", "2"),
+            ("1", "1.1"),
+            ("1", "1.2"),
+            ("2", "2.1"),
+            ("2", "2.2"),
+            ("2.1", "2.1.1"),
+            ("2.1", "2.1.2"),
+            ("2.2", "2.2.1"),
+            ("2.2", "2.2.2"),
+        ]
+    )
+
+
+@pytest.fixture
+def features_1d():
+    return np.array(
+        [
+            1,
+            2,
+            3,
+            4,
+            5,
+            6,
+            7,
+            8,
+            9,
+            10,
+            11,
+        ]
+    )
+
+
+@pytest.fixture
+def features_2d():
+    return np.array(
+        [
+            [1, 2],
+            [3, 4],
+            [5, 6],
+            [7, 8],
+            [9, 10],
+            [11, 12],
+            [13, 14],
+            [15, 16],
+            [17, 18],
+            [19, 20],
+            [21, 22],
+        ]
+    )
+
+
+@pytest.fixture
+def features_sparse():
+    return csr_matrix(
+        [
+            [1, 2],
+            [3, 4],
+            [5, 6],
+            [7, 8],
+            [9, 10],
+            [11, 12],
+            [13, 14],
+            [15, 16],
+            [17, 18],
+            [19, 20],
+            [21, 22],
+        ]
+    )
+
+
+@pytest.fixture
+def labels():
+    return np.array(
+        [
+            # labels that are the same as in the Single-Label Test case
+            [["1", "1.1"], ["", ""]],
+            [["1", "1.2"], ["", ""]],
+            [["2", "2.1"], ["", ""]],
+            [["2", "2.2"], ["", ""]],
+            [["2.1", "2.1.1"], ["", ""]],
+            [["2.1", "2.1.2"], ["", ""]],
+            [["2.2", "2.2.1"], ["", ""]],
+            [["2.2", "2.2.2"], ["", ""]],
+            # Multi-label Test cases
+            [["1", "1.1"], ["1", "1.2"]],
+            [["1", "1.1"], ["2", "2.1"]],
+            [["2.1", "2.1.1"], ["2.2", "2.2.2"]],
+        ]
+    )
+
+
+def test_binary_policy_positive_examples(digraph, features_1d, labels):
+    policy = BinaryPolicy(digraph, features_1d, labels)
+    with pytest.raises(NotImplementedError):
+        policy.positive_examples("1")
+
+
+def test_binary_policy_negative_examples(digraph, features_1d, labels):
+    policy = BinaryPolicy(digraph, features_1d, labels)
+    with pytest.raises(NotImplementedError):
+        policy.negative_examples("1")
+
+
+def test_exclusive_policy_positive_examples_1(digraph, features_1d, labels):
+    policy = ExclusivePolicy(digraph, features_1d, labels)
+    ground_truth = [
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        # Mutli-label Test cases
+        False,
+        False,
+        False,
```

Comment:
```suggestion
        True,
        True,
        False,
        False,
        False,
        False,
        False,
        False,
        # Mutli-label Test cases
        True,
        True,
        False,
```
I think we should deviate a little from the most specific definition here, otherwise all nodes close to the the root will have no positive training examples and will always return false. This in turn could lead to no predictions being made at all if there are no leaf nodes connected to the root node.